so_vector/challenges/default.json

```diff
@@ -0,0 +1,157 @@
+{
+  "name": "index-and-search",
+  "description": "Indexes vectors, then executes searches under various conditions",
+  "default": true,
+  "schedule": [
+    {
+      "operation": {
+        "operation-type": "delete-index"
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "create-index"
+      }
+    },
+    {
+      "operation": {
+        "operation-type": "cluster-health",
+        "request-params": {
+          "wait_for_status": "green"
+        },
+        "retry-until-success": true
+      }
+    },
+    {
+      "name": "index-append",
+      "operation": {
+        "operation-type": "bulk",
+        "bulk-size": {{bulk_size | default(500)}},
+        "ingest-percentage": {{ingest_percentage | default(100)}}
+      },
+      "warmup-time-period": {{ bulk_warmup | default(40) | int }},
+      "clients": {{bulk_indexing_clients | default(1)}}
+    },
+    {
+      "name": "refresh-after-index",
+      "operation": "refresh"
+    },
+    {
+      "name": "knn-search-10-50-match-all",
+      "operation": "knn-search-10-50-match-all",
+      "warmup-iterations": 100,
+      "iterations": 100,
+      "clients": 1
+    },
+    {
+      "name": "script-score-query-match-all",
+      "operation": "script-score-query-match-all",
+      "warmup-iterations": 100,
+      "iterations": 100,
+      "clients": 1
+    },
+    {
+      "name": "knn-search-10-50-acceptedAnswerId",
+      "operation": "knn-search-10-50-acceptedAnswerId",
+      "warmup-iterations": 100,
+      "iterations": 100,
+      "clients": 1
+    },
+    {
+      "name": "script-score-query-acceptedAnswerId",
+      "operation": "script-score-query-acceptedAnswerId",
+      "warmup-iterations": 100,
+      "iterations": 100,
+      "clients": 1
+    },
+    {
+      "name": "knn-search-10-50-java",
+      "operation": "knn-search-10-50-java",
```

Comment:
I wonder why do we need 3 queries for tags: `java`, css`, and `concurrency`? Do they represent filters of different restrictiveness ?

Reply:
Yes, these benchmark results show the different selectivity values: https://github.com/elastic/rally-tracks/pull/286#issuecomment-1197353545. It's roughly 7%, 2%, and 0.1%. If you think this is too much, maybe we could remove `ccs`.

Reply:
Thanks for clarification, nice that you have already tested for their selectivity.   I am ok with having all of them.